File: Dot examples/ASP Book Examples/Book Examples/Chapter34/vb/App_Code/Syndication/FeedHelper.vb
Imports System
Imports System.Web.Configuration
Imports System.ServiceModel.Syndication


''' <summary>
''' Summary description for FeedHelper
''' </summary>
Public Class FeedHelper

    Public Shared Function GetFeed() As SyndicationFeed

        Dim config As BlogSection = CType(WebConfigurationManager.GetSection("blog"), BlogSection)
        Dim feed As SyndicationFeed = New SyndicationFeed(config.Title, config.Description, New Uri(config.Url), Blog.SelectFeed())
        feed.BaseUri = New Uri(config.Url)
        Return feed
    End Function

End Class

File: Dot examples/ASP Book Examples/Book Examples/Chapter34/cs/App_Code/Syndication/FeedHelper.cs
using System;
using System.Web.Configuration;
using System.ServiceModel.Syndication;


/// <summary>
/// Summary description for FeedHelper
/// </summary>
public class FeedHelper
{

    public static SyndicationFeed GetFeed()
    {

        BlogSection config = (BlogSection)WebConfigurationManager.GetSection("blog");
        SyndicationFeed feed = new SyndicationFeed(config.Title, config.Description, new Uri(config.Url), Blog.SelectFeed());    
        feed.BaseUri = new Uri(config.Url);
        return feed;
    }

}
